backend/build.gradle.kts

```diff
@@ -252,6 +254,56 @@ tasks {
             into("./LegalDocML.de")
         }
     }
+
+    register("processDBFixtures") {
```

Comment:
This is simply so good! Having also XMLs for the seeds and letting gradle create the SQL. Amazing. I have just some observations, written in further comments